README.md

```diff
@@ -1,244 +1,28 @@
-
-### `latestEcmaVersion`
-
-Returns the latest ECMAScript supported by `espree`
-
-### `supportedEcmaVersions`
-
-Returns an array of all supported ECMAScript versions
-
-## Options
-
-```js
-const options = {
-    // attach range information to each node
-    range: false,
-
-    // attach line/column location information to each node
-    loc: false,
-
-    // create a top-level comments array containing all comments
-    comment: false,
-
-    // create a top-level tokens array containing all tokens
-    tokens: false,
-
-    // Set to 3, 5 (the default), 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 or 16 to specify the version of ECMAScript syntax you want to use.
-    // You can also set to 2015 (same as 6), 2016 (same as 7), 2017 (same as 8), 2018 (same as 9), 2019 (same as 10), 2020 (same as 11), 2021 (same as 12), 2022 (same as 13), 2023 (same as 14), 2024 (same as 15) or 2025 (same as 16) to use the year-based naming.
-    // You can also set "latest" to use the most recently supported version.
-    ecmaVersion: 3,
-
-    allowReserved: true, // only allowed when ecmaVersion is 3
-
-    // specify which type of script you're parsing ("script", "module", or "commonjs")
-    sourceType: "script",
-
-    // specify additional language features
-    ecmaFeatures: {
-
-        // enable JSX parsing
-        jsx: false,
-
-        // enable return in global scope (set to true automatically when sourceType is "commonjs")
-        globalReturn: false,
-
-        // enable implied strict mode (if ecmaVersion >= 5)
-        impliedStrict: false
-    }
-}
-```
-
-## Esprima Compatibility Going Forward
-
-The primary goal is to produce the exact same AST structure and tokens as Esprima, and that takes precedence over anything else. (The AST structure being the [ESTree](https://github.com/estree/estree) API with JSX extensions.) Separate from that, Espree may deviate from what Esprima outputs in terms of where and how comments are attached, as well as what additional information is available on AST nodes. That is to say, Espree may add more things to the AST nodes than Esprima does but the overall AST structure produced will be the same.
-
-Espree may also deviate from Esprima in the interface it exposes.
-
-## Contributing
-
-Issues and pull requests will be triaged and responded to as quickly as possible. We operate under the [ESLint Contributor Guidelines](http://eslint.org/docs/developer-guide/contributing), so please be sure to read them before contributing. If you're not sure where to dig in, check out the [issues](https://github.com/eslint/espree/issues).
-
-Espree is licensed under a permissive BSD 2-clause license.
-
-## Security Policy
-
-We work hard to ensure that Espree is safe for everyone and that security issues are addressed quickly and responsibly. Read the full [security policy](https://github.com/eslint/.github/blob/master/SECURITY.md).
```

Comment:
The security policy section is required to comply with one of Tidelift's recommendations. See https://tidelift.com/lifter/package/npm/espree/tasks/packages_have_security_policies.


![image](https://github.com/eslint/espree/assets/6367844/bfb0c238-4ef6-47bb-a978-e25d5e103e21)

We could change the link in Tidelift so it points to [packages/espree/docs/README.md](https://github.com/eslint/espree/blob/545d35b4ed76b02c14113d572da265738ae9d56f/packages/espree/docs/README.md#security-policy) instead, but I think it would be better to keep it pointing to a page that's not inside a package. Opinions?

Reply:
We should probably have it in both places: one that is Espree-specific in the Espree package and one at the repo root that is more generic.

Reply:
> We should probably have it in both places: one that is Espree-specific in the Espree package and one at the repo root that is more generic.

Sounds good. We could re-add the security policy to the main `README.md` file but replace "Espree" with something more generic (e.g. "packages in this repository"). I think that's the only change needed.

Reply:
Updated root Readme 👍🏻